manticore/platforms/linux.py

```diff
@@ -2621,8 +2629,34 @@ def solve_to_fd(data, fd):
             if name in ('_receive', '_read') and fd == 0:
                 solve_to_fd(data, inn)
 
+        first = True
+        argIO.write("[")
+        for a in self.argv:
+            if first:
+                first = False
+            else:
+                argIO.write(", ")
+            argIO.write("'")
+            solve_to_fd(a, argIO)
+            argIO.write("'")
+        argIO.write("]")
```

Comment:
Hey, a few comments about this specific implementation: This doesn't feel very pythonic. If summarizing like above, create an intermediate list with `solve_to_fd` results than just use `str.join` to create the final report. That, or even simpler, just produce a single argument per line since it's being produced to a separate file anyway.

Reply:
Also, if you can, add a unit test or two.

Reply:
> That, or even simpler, just produce a single argument per line since it's being produced to a separate file anyway.

Not sure I got it.
Is the new patch better ?
How would you handle arguments with multiple lines ?